Makefile.am

```diff
@@ -1171,6 +1167,8 @@ endif
 endif
 
 imap_lmtpd_SOURCES = \
+	imap/autocreate.c \
+	imap/autocreate.h \
```

Comment:
The .h files need to be mentioned in a single _SOURCES place in the whole Makefile.am .  The sole purpose of mentioning them ever, is to bundle them on `make dist`.